### ISSUE TITLE
[Client] WebSocket masking cannot be enabled

### ISSUE THREAD

**Issue by Lrns123** [ktorio/ktor#827]
### Ktor Version
1.1.0

### Ktor Engine Used(client or server and name)
CIO Client

### JVM Version, Operating System and Relevant Context
1.8_181

### Feedback
When using WebSockets (CIO Client engine), it is not possible to enable masking.

A code change introduced in response to #423 added an `override var masking: Boolean = true` to `DefaultClientWebSocketSession`, which overrides the `masking` property normally delegated to the underlying `RawWebSocket`, stopping it from updating the `WebSocketWriter`. 

By default, `RawWebSocket` has masking set to false, and since the property has been overriden, doing something like the following has no effect.
```
client.ws(....) {
  masking = true
  ...
}
```

**Comment by vngantk** [comment]
This is a bug. Instead of overriding the masking property by setting it to true, an init block should be used to initialise its value to true. See the following code:
```
class DefaultClientWebSocketSession(
    override val call: HttpClientCall,
    delegate: DefaultWebSocketSession
) : ClientWebSocketSession, DefaultWebSocketSession by delegate {
//    override var masking: Boolean = true     <-- bug
// Fix: initialise the property in an init block; this also preserves 
// the delegation to the underlying session delegate.
    init {
        masking = true
    }
}
```

**Comment by e5l** [comment]
Awesome :) merged. Thanks for the PR.

**Comment by cy6erGn0m** [comment]
Fixed in 1.1.1
